gitlab/objects.py

```diff
@@ -1249,6 +1249,18 @@ class ProjectKey(GitlabObject):
     requiredUrlAttrs = ['project_id']
     requiredCreateAttrs = ['title', 'key']
 
+    def enable(self, key_id, project_id):
```

Comment:
self should be the only argument, project and key id should be retrieved for the object attributes:
`````
url = '...' % (self.project_id, self.id)
`````